setup.py

```diff
@@ -36,6 +42,7 @@ def run(self):
     name="html2text",
     version=".".join(map(str, __import__('html2text').__version__)),
     description="Turn HTML into equivalent Markdown-structured text.",
+    long_description=read_md('README.md'),
```

Comment:
Nice so we can say #116 has been fixed.
Could you please update the ChangeLog for this fix as well ?